### ISSUE TITLE
Sync updates when initiated from background tasks

### ISSUE THREAD

**Issue by zph** [chrismccord/sync#123]
I'm filing this issue after banging my head on an issue for a good chunk of yesterday. (A followup from our chat on Twitter @chrismccord).

Background info: the sync behavior in my application works when models are updated within the Rails application. It also works as expected when updating a model from the Rails console. (Works as expected means: I can see the live sync behavior pushed out and displayed in the browser).

Where I'm stumped is how to get it working within a background task.  I'm currently using delayed_jobs to run tasks. I can verify in the rails console that the jobs are being created, saved, and updated as they go through various states (ie status moves from 'queued', 'processing', then 'success' / 'failed').

So the specific issue: 
- I must have a config setting wrong such that background jobs are not updating the page via the sync_new and sync partial calls.  The puzzling bit is that sync is working when the same models are created/saved/updated from other sources.

``` ruby
controllers/wiki_creations_controller.rb

class WikiCreationsController < ApplicationController
  before_action :set_wiki_creation, only: [:show, :edit, :update, :destroy]
  enable_sync only: [:create, :update, :destroy]
...
```

``` ruby
models/wiki_creation.rb

class WikiCreation < ActiveRecord::Base
  sync :all
  Sync::Model.enable! # Redundant?

...
```

``` haml
views/layouts/application.html.haml

!!!
%html
  %head
    %meta{:name => "viewport", :content => "width=device-width, initial-scale=1.0"}
    %title= content_for?(:title) ? yield(:title) : 'Wiki Gen'
    %meta{:name => "description", :content => "#{content_for?(:description) ? yield(:description) : 'Wiki Gen'}"}
    = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track' => true
    = javascript_include_tag 'application', 'data-turbolinks-track' => true
    = javascript_include_tag Sync.adapter_javascript_url
    = csrf_meta_tags
  %body
...
```

``` haml
views/projects/index.html.haml

...
%h2 Previous Wiki Generations
%table.table{style: "margin-top: 20px;"}
  %tr
    %th Name
    %th Wiki
    %th Repo
    %th Start Tag
    %th End Tag
    %th Release Date
    %th Requester
    %th Status

  = sync_new partial: 'wiki_creations_row', resource: WikiCreation.new
  = sync partial: 'wiki_creations_row', collection: @wiki_creations
```

Thoughts or troubleshooting tips are very welcome & thanks for pointing me in the right direction :).

tl;dr: What's the secret sauce for it working via background task?
